release_bot/utils.py

```diff
@@ -252,3 +253,17 @@ def update_version(file, new_version):
             output.write('\n'.join(content) + '\n')
         configuration.logger.info('Version replaced.')
     return changed
+
+def setupcfg_parser():
```

Comment:
```suggestion
def get_pypi_project_name_from_setup_cfg():
```
this name is much more descriptive